woocommerce/woocommerce-gutenberg-products-block
# Audit and Move Image Assets, update error state

This is part of https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/4299

I went through all images assets and removed the unused ones, and moved the ones we need to keep around (that includes payment icons, some preview images, and the error icon).

- I was able to remove `avatar.png` because we can use the default avatar from WordPress.
- I inlined the block preview SVGs instead of linking.
- I made some small changes to `error-state.svg` to simplify it and reduce its size. Mostly just cleaned up some paths and recoloured based on the a8c studio pallette.
- I took the liberty of styling the error state a little better. This state isn't visible often but is user facing so I made it a little more friendly and made the CTA button more obvious. Screenshot below.

### Screenshots

This is the error state styling after this patch:

![Screenshot 2021-06-23 at 11 24 29](https://user-images.githubusercontent.com/90977/123083420-f2eeba80-d417-11eb-88be-8a0f9b778cc5.png)

### How to test the changes in this Pull Request:

1. Tests should pass (updated snaps)
2. Build should succeed
3. In the editor, view the cart and checkout blocks. Ensure preview images are still present.
4. Repeat for review block. Should still be an avatar image.
5. You can [force an error here](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/343f70d7af57f751c327349dec864f9e176d0f70/assets/js/base/components/block-error-boundary/index.js#L46) and view the cart page.
